### PR TITLE
List of comments on articles has been added

### PR DESCRIPTION
**Pull request by alperenkozkan**
# Pull Request Template

## Description

Please include a summary of the work done and which issue is addressed.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (new functionality)
- [ ] This change requires a documentation update(like wiki/requirements or design update)

## How Has This Been Tested?

Please describe the tests that you ran. 
If you haven't committed related tests, please share the related issue requesting tests for this functionality.

Test Issue # (issue)

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation, or created a related issue.
- [ ] I have committed tests that prove my fix is effective or that my feature works. Or, I have opened an issue.
- [ ] I have added 2 reviewers to get the approval of.

### CODE REVIEW

File: traiders/frontend/public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Traiders</title>
+    <title>React App</title>
```
Comment (furkangok) [inline traiders/frontend/public/index.html]: I think you made this commit without pulling the last version of the master, try to pull master branch into your branch and then make another commit. The problem with this commit is it is trying to remove some changes done by other team members.

File: traiders/frontend/src/components/comment/comment.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
+.comment-container {
+  width: 65%;
```
Comment (furkangok) [inline traiders/frontend/src/components/comment/comment.scss]: I think there should be some lower width limit.

File: traiders/frontend/public/index.html (hunk 1 above)
Comment (alperenkozkan) [reply traiders/frontend/public/index.html]: It should be fixed
